localisation/000_Other_title_names.csv

```diff
@@ -304,7 +304,7 @@ d_voidfarer_adj;Void;Vide;;;Vac
 d_forsaken_cult;Cult of Forgotten Shadows;Culte des Ombres oubli�es;Kult des vergessenen Schattens;;;;;;;;;;;x
 d_forsaken_cult_adj;Shadow;Ombre-oubli�e;;;;;;;;;;;;x
 PROV132;Helm;Helm;;;;;;;;;;;;x
-d_lightbound;The Lightbound;Empreint de Lumi�re;Der Lichtgebundene;;Vinculada a la Luz;;;;;;;;;x
+d_lightbound;The Lightbound;Empreint de Lumi�re;Die Lichtgebundenen;;Vinculada a la Luz;;;;;;;;;x
```

Comment:
@arithon why you kept "the" here?

Reply:
Dunno, this was probably changed before I knew about the "the"-problem with religious heads. Changed it now